tests/template_tests/filter_tests/test_escapeseq.py

```diff
@@ -0,0 +1,51 @@
+from django.test import SimpleTestCase
+from django.utils.safestring import mark_safe
+
+from ..utils import setup
+
+
+class EscapeSeqTests(SimpleTestCase):
```

Comment:
Felix: I almost wonder whether it's worth putting these alongside the escape tests🤔

Reply:
I think it's worth keeping the current pattern with each template filter having a separate file.